dali_tf_plugin/daliop.cc

```diff
@@ -101,7 +101,7 @@ class DaliOp : public tf::OpKernel {
 
     int num_threads;
     int device_id;
-    int batch_size;
+    int max_batch_size;
```

Comment:
@awolant Is it possible to have a variable batch size in Tensorflow?

Reply:
Yes, there are ways to write a graph that can have variable batch size. You can do something like:
```
inputs = tf.placeholder(tf.float32, [None, 32, 32, 3])
batch_size = tf.shape(inputs[0])[0] 
```
Here `batch_size` is a tensor and the value will be known during the execution only.

For newer APIs (like tf.data, and TF 2.x in general) it's not so easy to do, but the `daliop` integrates with TF 1.x and graph approach, so I think it is reasonable to make it possible from DALI side.